project/src/components/offers-list/offers-list.tsx

```diff
@@ -1,31 +1,27 @@
-import CityCard from '../../components/city-card/city-card';
+import OfferCard from '../offer-card/offer-card';
 import { Offer } from '../../types/Offer';
-import { useState } from 'react';
 
 type OffersListProps = {
   offers: Offer[];
+  onActiveCardId?: (id: number | null) => void;
 }
 
-const OffersList = ({ offers }: OffersListProps): JSX.Element => {
-  const [, setActiveCardId] = useState<number | null>(null);
-
-  return (
-    <div className="cities__places-list places__list">
-      {
-        offers.map((offer) =>
-          (
-            <CityCard
-              key={offer.id}
-              onActiveCardId={setActiveCardId}
-              offer={offer}
-              classNameWrapper="cities__image-wrapper"
-              className="cities__card"
-            />
-          )
+const OffersList = ({ offers, onActiveCardId }: OffersListProps): JSX.Element => (
+  <div className="cities__places-list places__list">
+    {
+      offers.map((offer) =>
+        (
+          <OfferCard
+            key={offer.id}
+            onActiveCardId={onActiveCardId}
+            offer={offer}
+            classNameWrapper="cities__image-wrapper"
+            className="cities__card"
+          />
         )
```

Comment:
Форматируют обычно так 
```suggestion
      offers.map((offer) =>  (
          <OfferCard
            key={offer.id}
            onActiveCardId={onActiveCardId}
            offer={offer}
            classNameWrapper="cities__image-wrapper"
            className="cities__card"
          />)
```